spec/serializers/federation_entity_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe FederationEntitySerializer do
+  class TestEntity < DiasporaFederation::Entity
+    property :test, :string
+  end
+
+  let(:object) { double }
+
+  def enable_test_entity_builder
+    allow(Diaspora::Federation::Mappings).to receive(:builder_for).with(object.class).and_return(:test_entity)
+    allow(Diaspora::Federation::Entities).to receive(:test_entity).with(object) {
+      TestEntity.new(test: "asdf")
+    }
+  end
+
+  before do
+    enable_test_entity_builder
+  end
+
+  it_behaves_like "it is federation entity serializer"
```

Comment:
Nit: "a federation entity serializer", results in:
* `shared_examples_for "a federation entity serializer"`
* `it_behaves_like "a federation entity serializer"`